lib/graphql-bookshelf-bridge/util/initDataLoaders.js

```diff
@@ -23,7 +23,9 @@ export default function initDataLoaders (spec) {
   // general-purpose query cache, for relational SQL queries that aren't just
   // fetching objects by ID.
   loaders.queries = new DataLoader(
-    queries => Promise.map(queries, q => q.fetch()),
+    queries => Promise.map(queries, ({ query, method }) => {
+      return method ? query[method]() : query.fetch()
```

Comment:
this here works around something in Bookshelf's API that I have always thought of as a wart: when you have a relation, `fetch` gets the whole set and you use `fetchOne` to get a single row, but when you have a collection built from `Post.query(q => { ... })`, `fetch` gets a single result and `fetchAll` gets the whole set. We were actually relying on the ambiguity of `fetch` here to handle both `fetchRelation` and `fetchOne`, but now that we're using `Search`, we can't do so anymore.